exampleSite/config.toml

```diff
@@ -14,7 +14,6 @@ theme = "../.."
 #     featuredpost = "true"
 
 [params.settings]
-    googleanalytics = "UA-113904582-3"
```

Comment:
I wonder if we should leave this here but commented out.

The example site is a good way of other developers knowing what the theme provides.

Thoughts?